<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>本地对战</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stratBtn_web">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>局域网对战</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>955</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>选项</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>新的对局</string>
     </property>
     <addaction name="actionloCAL"/>
     <addaction name="actiononline"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menuNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>支持</string>
    </property>
    <addaction name="actionabout"/>
    <addaction name="actionsupport"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRestart">
   <property name="text">
    <string>Restart</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出游戏</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>读取存档</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>学生信息</string>
   </property>
  </action>
  <action name="actionsupport">
   <property name="text">
    <string>联系作者</string>
   </property>
  </action>
  <action name="actionloCAL">
   <property name="text">
    <string>本地对战</string>
   </property>
  </action>
  <action name="actiononline">
   <property name="text">
    <string>网络对战</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
